routes/web.php

```diff
@@ -14,3 +14,12 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
+Route::group(['namespace' => 'Admin', 'prefix' => 'admin'], function(){
+	Route::resource('/news', 'NewsController');
+	Route::resource('/category','CategoryController');
```

Comment:
route endpoints should be in plural, not single. `/categories`, `routes`
http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api